<div style="display:inline-block; position:absolute; top:0; right:0; color: #230e3e; font:italic 1em serif;">$\LaTeX$<span style="display:inline-block; vertical-align:bottom"><a href="https://www.mathjax.org">
    <img style="height:1.25em" title="Powered by MathJax"
    src="https://www.mathjax.org/badge/mj_logo.png"
    border="0" alt="Powered by MathJax" />
</a></span> </div>

<header style="background-color:#e7e2ee; color:#461d7c; line-height:2em; padding-left:1em; padding-right:1em; bottom:0; float:bottom; text-align:right; float:right; font-family:serif; font-size:small;">Problem 4-34 ⋮ Çengel, Y. A., and Boles, M. A. (2001). <i>Thermodynamics: An Engineering Approach.</i> Boston: McGraw-Hill.</header>  




### Given   
An insulated piston–cylinder device contains 5 L of saturated liquid water at a constant pressure of 175 kPa. Water is stirred by a paddle wheel while a current of 8 A flows for 45 min through a resistor placed in the water. One-half of the liquid is evaporated during this constant-pressure process and the paddle-wheel work amounts to 400 kJ. 


### Find 
1. Determine the voltage of the source.  
1. Show the process on a P-V diagram with respect to saturation lines. 

### Get Organized

Even dysfunctional homes deserve a decent diagram.

![figure, cb8, 4-34](figures/ch04_cen98179_p04034_lg.jpg)

#### This gets easy if you recognize these things:  
1. Closed system ⋮ constant mass  
1. Insulated ⋮ no heat transfer   
1. Isobaric process in closed system ⋮ expect $\Delta V$ and the boundary work that comes with it.  
1. We can resolve both states ⋮ $(P_1,x_1),\ (x_2,P_2=P_1)$  
1. We can assume $\rm \Delta KE = \Delta PE \approx 0$  
1. Don't forget: energy transfer that isn't heat transfer is all classified as work.

#### Assumptions
1. $\rm \Delta KE = \Delta PE \approx 0$   
1. Volume will increase. If we're wrong, we'll see it when we calculate boundary work.



<div style="font-size:4000%; font-weight:900; line-height:1em; background-color:#461d7c; color:#fdd023; text-align:center;">Attack</div>

A few more things:

1. We are given $W_\textit{paddle,in} = 400\,\text{kJ}$  
2. Recall, $W_\textit{electricity} = \left[ \dot W_{\substack{\textit{electricity} \\ \textit{(power)}} }\right]\textit{time} = \text{VI}t$
3. $W_{b,out} = \displaystyle \int^2_1 P\,{\rm d}V$

We'll solve for the electrical work, then break that down to get the voltage we need. You can solve directly if you prefer.  
Apply the First Law, with subscripts 1 and 2 representing initial and final states, respectively:

\begin{array}{ccccc}
  &E_{in} &&-& E_{out} &=& \Delta E_{sys} \\
  W_\textit{electricity,in} &+& W_\textit{paddle,in} &-& W_{b,out} &=& U_2 - U_1 \\ 
  W_\textit{electricity,in} &+& 400\,\text{kJ} &-& \displaystyle \int^2_1 P\,{\rm d}V &=&  U_2 - U_1 \\[6pt]{}
\end{array}


\begin{align*}
   W_\textit{electricity,in} &= U_2 - U_1 + \int^2_1 P\,{\rm d}V - 400\,\text{kJ} \\[6pt]
     &= U_2 - U_1 + P\left( V_2 - V_1 \right) - 400\,\text{kJ} \\[9pt]
     &= \left( U_2 + P V_2\right) - \left(U_1 + P V_1 \right) - 400\,\text{kJ} \\[9pt]{}
     &= H_2 - H_1 - 400\,\text{kJ} \\[9pt]{}
\end{align*}


\begin{gather*}
  W_\textit{electricity,in} = m\left( h_2 - h_1 \right) - 400\,\text{kJ} \\[6pt]
  m = \frac{V_1}{v_1}  \qquad v_1 = v \left(P_1, x_1 \right) 
  \qquad h_1 = h \left(P, x_1 \right) \qquad  h_2 = h \left( P, x_2 \right)  \\
\end{gather*}

In [1]:
using Unitful

In [2]:
include("src/marcucius.jl");

running marcucius...
package LaTeXStrings...package Unitful...function roundme...function sopretty...function subscripts...macro sopretty...macro rubber...marcucius done!


In [3]:

#___ given____________
V = 5u"L"
I = 8u"A"
time = 45u"minute"
x_1 = 0
x_2 = 0.5
W_paddle = 400u"kJ"

v_f_170kPa  = 1.0528e-3u"m^3/kg" 
v_f_180kPa  = 1.0605e-3u"m^3/kg" 

h_f_170kPa  = 467.11u"kJ/kg"  
h_f_180kPa  = 504.70u"kJ/kg" 
h_fg_170kPa = 2226.5u"kJ/kg"
h_fg_180kPa = 2201.9u"kJ/kg" 

v_f_175kPa  = roundme((v_f_170kPa  + v_f_180kPa)/2, 10)
h_f_175kPa  = (h_f_170kPa  + h_f_180kPa)/2
h_fg_175kPa = (h_fg_170kPa + h_fg_180kPa)/2


m = roundme((V / v_f_175kPa)|> u"kg") 
h_1 = h_f_175kPa  
h_2 = (h_f_175kPa + x_2 * h_fg_175kPa)   |> roundme

W_elec_in = (m * (h_2 - h_1) - W_paddle) |> roundme

voltage = W_elec_in / (time * I) |> u"V" |> roundme

(@sopretty v_f_175kPa  render left fraction)    |> display
(@sopretty h_f_175kPa  render left fraction)    |> display
(@sopretty h_fg_175kPa render left fraction)    |> display
(@sopretty m render left fraction)              |> display
(@sopretty h_1 render left fraction)            |> display
(@sopretty h_2 render left fraction)            |> display
(@sopretty W_elec_in render left fraction)      |> display

(@sopretty voltage render left fraction display)|> display



L"$v_{ \text{f,175kPa}} = \textstyle 0.00105665~\frac{\rm  m^{3} }{\rm  kg }$"

L"$h_{ \text{f,175kPa}} = \textstyle 485.905~\frac{\rm  kJ }{\rm  kg }$"

L"$h_{ \text{fg,175kPa}} = \textstyle 2214.2~\frac{\rm  kJ }{\rm  kg }$"

L"$m = \textstyle 4.7319~{\rm  kg }{\rm  }$"

L"$h_{ \text{1}} = \textstyle 485.905~\frac{\rm  kJ }{\rm  kg }$"

L"$h_{ \text{2}} = \textstyle 1593.005~\frac{\rm  kJ }{\rm  kg }$"

L"$W_{ \text{elec,in}} = \textstyle 4838.6865~{\rm  kJ }{\rm  }$"

L"$voltage = \displaystyle 224.0133~{\rm  V }{\rm  }$"

<div class="btn-success">&nbsp;<div class="badge"></div><i class="fa fa-bomb"></i><strong> Boom</strong></div>
<div class="panel-success"><div class="panel-heading"><div class="badge"></div> 

\begin{equation*}
   224\,\text{Volts}
\end{equation*}  


</div></div>

<footer style="background-color: #461d7c; color:rgba(253,208,35,0.35); line-height:2em; padding-left:1em; padding-right:0.5em; bottom:0; float:bottom; text-align:right;"><i>ψᵟ</i></footer>